### PR TITLE
TVAULT-5505 OSA remove taskflow_max_age variable from triliovault-wlm.conf  templates

### PR DESCRIPTION
**Pull request by trilio-shiwanand**
Removed the taskflow_max_age variable from the trilioault-wlm.conf file

### CODE REVIEW

File: ansible/roles/openstack-ansible-os_triliovault_wlm/defaults/main.yml [hunk 1]
```diff
@@ -198,14 +198,12 @@ NOVA_GID: 998
 barbican_encryption: "True"
 
 
+## TASKFLOW PARAM
```
Comment (dhiraj-trilio) [inline ansible/roles/openstack-ansible-os_triliovault_wlm/defaults/main.yml:201]: Why is this in caps ?

Comment (trilio-shiwanand) [reply ansible/roles/openstack-ansible-os_triliovault_wlm/defaults/main.yml:201]: it's a comment only

Comment (dhiraj-trilio) [reply ansible/roles/openstack-ansible-os_triliovault_wlm/defaults/main.yml:201]: see below lines

Comment (trilio-shiwanand) [reply ansible/roles/openstack-ansible-os_triliovault_wlm/defaults/main.yml:201]: @dhiraj-trilio these are the vars kind of placeholders.

Comment (dhiraj-trilio) [reply ansible/roles/openstack-ansible-os_triliovault_wlm/defaults/main.yml:201]: can we keep in lower case just like other variables in this file eg. barbican_encryption, wlm_log_dir ? ideally we place all variables in conf file in lowercase letter only. This file is mixed of upper and lower case variables. can we keep all these in lowercase ?

Comment (trilio-shiwanand) [reply ansible/roles/openstack-ansible-os_triliovault_wlm/defaults/main.yml:201]: @dhiraj-trilio yes will do it in 5.0 SP1

**Review by dhiraj-trilio — APPROVED** [review]

**Review by shyam-biradar — APPROVED** [review]

**Pull request merged**